### PR TITLE
[FSPE-6923] Created 3rd level hierarchy in data table

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSPE-6923

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://dd49a3de86baed3e5d97777e01e0914f25153dd9-chi-pr.enterprise.ctl.io" target="_blank"> https://dd49a3de86baed3e5d97777e01e0914f25153dd9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1698/ for more details.

**Review by mattnickles — APPROVED** [review]

File: src/website/views/components/data-table/_examples.pug [hunk 1]
```diff
@@ -4780,6 +4785,11 @@ h3 Portal -- Accordion Expanded -- Child Expanded
               <div class="chi-data-table__row-grand-child">
                 <div class="chi-data-table__cell -expandable"></div>
                 <div class="chi-data-table__cell -key -bold">
+                  <button class="chi-button -icon -flat -sm -expand" aria-label="Expand row">
```
Comment (janibolkvadze) [inline src/website/views/components/data-table/_examples.pug:4788]: Looks like the code snippet is a bit off. I think the button is supposed to be on row-child level.

Screenshot 2021-01-18 at 12 46 55

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: src/website/views/components/data-table/_examples.pug (hunk 1 above)
Comment (jllr) [reply src/website/views/components/data-table/_examples.pug:4788]: @janibolkvadze good catch!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://36434578d35ba65310695fbf24f96b6a09515e65-chi-pr.enterprise.ctl.io" target="_blank"> https://36434578d35ba65310695fbf24f96b6a09515e65-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1702/ for more details.

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-e4nx77, https://ops.elasticbox.com/services/instances/i-7ctmom

**Pull request merged**